### PR TITLE
Memberships - remove from simple payments

### PR DESCRIPTION
**Pull request by artpi**
This removes memberships from simple payments and puts in its own form.

### CODE REVIEW

File: client/components/data/query-memberships/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+	UNSAFE_componentWillMount() {
+		this.request( this.props );
+	}
+
+	UNSAFE_componentWillReceiveProps( nextProps ) {
+		if ( nextProps.siteId !== this.props.siteId || nextProps.productId !== this.props.productId ) {
+			this.request( nextProps );
+		}
+	}
```
Comment (dmsnell) [inline client/components/data/query-memberships/index.js]: the point of `USAFE_` is for existing code, not new code 😉

Comment (artpi) [reply client/components/data/query-memberships/index.js]: And that is why it was not under review :)

Comment (artpi) [reply client/components/data/query-memberships/index.js]: That whole PR has to be refactored once someone actually puts resources on this project. Unfortunately, there are none and the clock is ticking :/

Comment (artpi) [reply client/components/data/query-memberships/index.js]: But I am writing a huge "what is all this memberships code" post explaining the situation, so hang tight :)

Comment (dmsnell) [reply client/components/data/query-memberships/index.js]: > And that is why it was not under review :)

let's not merge it then 👍

Comment (artpi) [reply client/components/data/query-memberships/index.js]: Recitified in https://github.com/Automattic/wp-calypso/pull/26022

**Pull request merged**